### ISSUE TITLE
LLVM 11.1.0 recipe

### ISSUE THREAD

**Issue by volo-droid** [haikuports/haikuports#6209]
There's LLVM 12 recipe already. Can a recipe for LLVM 11.1.0 be added too? I'm trying to hack on some stuff which depends on 11.1.0 . I'm happy to add it myself, but would probably need an assistance there.

**Comment by pulkomandy** [comment]
It looks like llvm comes with more tools now :)

You can add them to the PROVIDES.

Make sure to move the libraries to the development package (adjust prepareInstalledDevelLibs as needed) and add them to PROVIDES_devel.

**Comment by threedeyes** [comment]
I also found that in all versions of llvm, the paths to libraries and include files in cmake files were incorrectly specified. When building projects, static libraries are at least looked for in /system/lib, and not in /system/develop/lib. Headers files are looked for in /system/include.

**Comment by waddlesplash** [comment]
> I also found that in all versions of llvm, the paths to libraries and include files in cmake files were incorrectly specified. When building projects, static libraries are at least looked for in /system/lib, and not in /system/develop/lib. Headers files are looked for in /system/include.

This has its own report: #3749.

I tried to fix this some time ago but I think I got bogged down in other things.

**Comment by korli** [comment]
Do you still need LLVM 11 ?

**Comment by volo-droid** [comment]
> Do you still need LLVM 11 ?

Yes, I still do. Unfortunately, I didn't finish my work on the recipe back in 2021. Recently I got back to tinkering with Haiku, and still have the VM with the changes available but would need to replace the SSD holding it, as it has dangerously degraded according to the SMART info.

**Comment by volo-droid** [comment]
It's time to close this one, as newer LLVM versions are available for some time, and I don't need 11.1.0 anymore.
